include/kinsol/kinsol.h

```diff
@@ -84,9 +84,6 @@ extern "C" {
 
 typedef int (*KINSysFn)(N_Vector uu, N_Vector fval, void* user_data);
 
-typedef void (*KINInfoHandlerFn)(const char* module, const char* function,
-                                 char* msg, void* user_data);
-
 typedef int (*KINDampingFn)(long int iter, N_Vector u_val, N_Vector g_val,
```

Comment:
Do parameters in `KINDampingFn` and `KINDepthFn` need `_1d` and `_out` suffixes (i.e., `qt_fn_1d`, `damping_factor_out`, `df_1d`, `R_mat_1d`, `new_depth_out`, and `remove_indices_1d`)